ts/features/bonus/cgn/components/merchants/discount/CgnBucketCodeComponent.tsx

```diff
@@ -62,46 +62,68 @@ const CgnBucketCodeComponent = () => {
     []
   );
 
+  if (isLoading(code)) {
+    return <ActivityIndicator />;
+  }
+
+  // we got an error no code is available
+  if (isError(code)) {
+    return null;
+  }
```

Comment:
What do you think about managing the error case also in `CgnMerchantsDiscountItem.tsx` showing an error message instead of the bottomsheet?
If I understand well without the code the bottomsheet content is useless.

Reply:
yes, indeed I should add a TODO since there's an appropriate task on jira https://pagopa.atlassian.net/browse/IOACGN-21